ci-operator/config/openshift/cluster-version-operator/openshift-cluster-version-operator-master.yaml

```diff
@@ -68,6 +68,14 @@ tests:
   steps:
     cluster_profile: gcp
     workflow: openshift-upgrade-gcp-loki
+- as: e2e-rollback
```

Comment:
I don't think we need a separate test, when we can just add:

```yaml
env:
  TEST_OPTIONS: abort-at=100
```

to the existing `e2e-upgrade` entry.

Reply:
I think we want these separate. `e2e-upgrade` won't attempt to do a rollback, as it may flake too often. 
`e2e-rollback` tests are optional and can be skipped (we might want to land some important fix and sort out rollback failures later)

Reply:
I'd rather `/override ...` when we need to break rollback

Reply:
Ah, I never noticed we always had rollback in `e2e-upgrade`. Re-added the env to `e2e-agnostic-upgrade`